Makefile

```diff
@@ -82,9 +82,8 @@ ifndef HAS_YQ
 endif
 
 ifndef HAS_KUSTOMIZE
```

Comment:
Do we still need this in the Makefile if we're using kubectl?

Reply:
yes, I removed them, thanks for the reminder